demisto_sdk/commands/pre_commit/README.md

```diff
@@ -33,6 +33,27 @@ You can set this as follows.
 And call precommit as follows: `demisto-sdk pre-commit -a --mode nightly`.
 Note, that it is possible to use any mode that you like, and have multiple modes for each hook, like in the example.
 
+## Hooks
+Hooks can be set in the `.pre-commit-config_template.yaml` file. The syntax is similar to the official [`pre-commit` hooks](https://pre-commit.com/#new-hooks). But the command allows more keys to be set:
+
+### Skip key
+In order to skip certain hook from running, you can add a `skip` key to the hook configuration.
+```yaml
+- id: sample-hook
+  skip: true
+```
+This key could be use together with [mode](#modes), to skip certain hook when running in a specific mode.
+
+# Needs key
+Needs keys allow to define dependencies between hooks. If a hook with `needs` is skipped, hooks that depend on it will also be skipped.
```

Comment:
needs doesnt guarantee order? are you sure we dont want to just make them use skip:true for this? Seems like we're complicating things

Reply:
oops